src/clj/game/cards/resources.clj

```diff
@@ -993,7 +993,9 @@
                                                                   (count from) from) card nil)
                      (do (clear-wait-prompt state :corp)
                          (effect-completed state side eid card)))))
-    :trash-effect {:effect (effect (mill :runner 3))}}
+    :trash-effect {:msg (msg "trash " (join ", " (map :title (take 3 (:deck runner))))
+                             " from their Stack as Rolodex was trashed ")
```

Comment:
Now reads: "wozzit trashes Rumor Mill, Employee Strike, Rolodex from their Stack due to Rolodex being trashed."  Just read the original card 5 times and see if it makes sense :p